openapi-cli/src/test/java/io/ballerina/openapi/generators/schema/EnumGenerationTests.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Updated in https://github.com/ballerina-platform/openapi-tools/pull/1328/commits/1c1b409182ccd309a4da54dd57c18e65241dd1bc